README.md

```diff
@@ -1,48 +1,66 @@
+- The user can disable cells in the garden.
+![](https://user-images.githubusercontent.com/126428377/265612258-c92f3e0d-3f4c-4b31-88c6-f99b477c95b2.gif)
+- The user can search for plants to add to their garden.
+![](https://user-images.githubusercontent.com/126428377/265613058-e701ab11-d143-4f5f-9beb-0de94d5786b6.gif)
+- Quantities of new and existing plants in the garden are continuously updated.
+![](https://user-images.githubusercontent.com/126428377/265614891-c454c201-7d52-4819-b8ff-4389420b52b4.gif)
+- The user can view more information about any particular plant.
+![](https://user-images.githubusercontent.com/126428377/265615593-2369b048-84e8-42ce-be3c-259a5a8e1b87.gif)
+- The user can remove unplanted items from their garden.
+![](https://user-images.githubusercontent.com/126428377/265617067-60b845b6-554e-45f5-acab-e49e1f8f36d3.gif)
+- The user can reset their garden and start fresh.
+![](https://user-images.githubusercontent.com/126428377/265617509-f67b13e8-b354-4ec4-8b31-c75325c52f21.gif)
+
+## SET UP
+- Fork this repository.
+- Copy the SSH key in order to clone your forked repository down into your local device.
+- `cd` into the cloned directory.
+- Run `npm install` to install project dependancies.
+- Run `npm start` in the terminal to see the HTML page
+  - `Control + C` is the command to stop running the local server.
+
+## CONTRIBUTERS
+- Front End:
+  - Daniel Cochico: [GitHub](https://github.com/dcochico) | [LinkedIn](https://www.linkedin.com/in/daniel-c-3a0201176/)
+  - Matt Fowler: [GitHub](https://github.com/mbenfowler) | [LinkedIn](https://www.linkedin.com/in/mbenfowler/)
+  - Alexa VanSchaardenburg: [GitHub](https://github.com/AlexaVanSchaardenburg) | [LinkedIn](https://www.linkedin.com/in/alexa-r-vanschaardenburg/)
+<br>
+- Back End:
+  - Kim Bergstrom: [GitHub](https://github.com/kbergstrom78) | [LinkedIn](https://www.linkedin.com/in/kimberley-bergstrom/)
+  - Boston Lowrey: [GitHub](https://github.com/BLowrey24) | [LinkedIn](https://www.linkedin.com/in/boston-lowrey/)
+  - Myles Nottingham: [GitHub](https://github.com/MylesNottingham) | [LinkedIn](https://www.linkedin.com/in/mylesnottingham/)
+  - Austin Woods: [GitHub](https://github.com/boomclear) | [LinkedIn](https://www.linkedin.com/in/austin-woods-1830aa195/)
+
+## CONTEXT
+- Final Capstone project as part of the [Turing School of Software & Design](https://turing.edu/) program.
+
+## LEARNING GOALS
+- Explore and implement TypeScript concepts that have not been explicitly taught while at Turing
+- Gain experience using continuous integration tools to build and automate the deployment of features
+- Focus on communication between front-end and back-end teams
+
+## FUTURE EXTENSIONS
+- The user should receive timely alerts for watering plants in their garden.
+- The user should be able to input their location and be shown what hardiness zone they are in.
+- The user should be able to customize the dimensions of their garden grid.
```

Comment:
i'd put something around individual user login.
Also I'd love to put here something along the lines of a user having multiple plots they can manage!